SokolTextGame/Commands.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SokolTextGame
+{
+    internal interface ICommand
+    {
+        void Execute(string[] words, World world);
+                
+    }
+    public class Echo : ICommand
+    {
+        void ICommand.Execute(string[] words, World world)
+        {
+            string text = string.Join(" ", words).Substring(5);
```

Comment:
It's bad to hardcode the number of symbols like this (the 5 because of "echo" plus space). Use string.Join(), but with words.Skip(1) instead. This way, the joined string will exclude the first word no matter how long it is.